src/posthog-core.ts

```diff
@@ -1826,6 +1825,19 @@ export class PostHog {
         )
     }
 
+    /**
+     *  Enables person processing for the current user (future events will create a Person profile) when
+     *  config.person_profiles is set to 'identified_only'. Produces a warning if config.person_profiles is set to
+     *  'never'.
+     */
+    enablePersonProfile(): void {
```

Comment:
should this be `createPersonProfile()` instead of `enable`? That language makes more sense in my brain.

Reply:
I find it weird that it sends an event tbh. Does it need to? Can it not just update the config setting so that the next event creates a profile?

Reply:
I debated this, but I think this would be very confusing if there were no further events, and you therefore didn't end up with a profile despite calling `createPersonProfile`.

Reply:
Remember that $pageleave is off by default and flaky on mobile